ovos_plugin_common_play/ocp/player.py

```diff
@@ -468,23 +469,26 @@ def play_shuffle(self):
 
     def play_next(self):
         """
-        Play the next track in the playlist.
-        If there is no next track, end playback.
+        Play the next track in the playlist or search results.
+        End playback if there is no next track, accounting for repeat and
+        shuffle settings.
         """
-        if self.active_backend in [PlaybackType.MPRIS]:
```

Comment:
this was planning for expansion, want to support more protocols other than MPRIS in the future

Reply:
Makes sense; looks like this is consistent in other methods too so I'll revert this change